docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -511,16 +521,191 @@ To manage the media controller playlist in your application, you must handle req
 
 3. Register the listener to start receiving notifications about playlist changes:
 
-   ```
+   ```javascript
    watcherId = mcServerInfo.addPlaylistUpdatedListener(listener);
    ```
 
 4. To stop receiving notifications, use the `removePlaylistUpdatedListener()` method:
 
-   ```
+   ```javascript
    mcServerInfo.removePlaylistUpdatedListener(watcherId);
    ```
 
+## Media Controller Server Abilities
+
+Various abilities can be set to give the information to the clients about the supported features. These abilities can be divided into two groups:
+
+1. Media controller simple abilities
+
+    Each ability is described by a single [MediaControllerAbilitySupport](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerAbilitySupport) value and is not a part of a complex ability structure:
```

Comment:
Question. It looks like MediaControllerAbilitySupport is supported by mobile, wearable, and TV.
However in here we have link to mobile only. Don't we need to add links to wearable and TV also?

Reply:
MediaControllerAbilitySupport type is identical for all profiles, which was the reason to link this only one time. for better readibility in final view, but if you do not agree with this justification, links for all profiles could be provided.